Nonprofit-Exchange-Hub/web-app
# HTTP response logger middleware

## Dev Summary

This PR adds an HTTP response middle ware. It's part of the ticket. 

#### Not implemented
Log levels for different environments




## Test Plan


repro steps:

1.  start the server
2. Make any http request to an `/api` endpoint
3. Expect to see the http response logged
4. Expect to NOT see the static assets http requests logged. In other words, when you navigate through the front end, only the requests to the `api/` route is logged 


## Resources

n/a
